chain-impl-mockchain/src/leadership/genesis/mod.rs

```diff
@@ -11,7 +11,7 @@ use crate::{
 };
 use chain_crypto::Verification as SigningVerification;
 use chain_crypto::{Curve25519_2HashDH, FakeMMM, PublicKey, SecretKey};
-pub use vrfeval::Witness;
+pub use vrfeval::{FError, Witness, F};
```

Comment:
Is it really denoted only as F in our literature?

Reply:
I belive so, at least here: https://hydra.iohk.io/build/796650/download/1/non-integer-calculations.pdf It's also consistently called "F" throughout the codebase.

Reply:
Ouroboros Praos paper describes `f` as `active slots coefficient`. This is an awkward situation, that's a much better name, both in config files and in codebase, but the change will be big. Is it worth considering?

Reply:
Maybe it's worth it. the clarity of reading "active-slots-coefficient: 0.1" compared to "f: 0.1" in config file and source code is unarguable.

Reply:
Ok, next PR then :)

Reply:
Added in this PR

Reply:
_(bikeshedding hat on:)_ `active-slots-ratio`?